src/frontend/src/flows/manage/index.ts

```diff
@@ -297,24 +322,53 @@ export const displayManage = (
         "More than one recovery keys are registered, which is unexpected. Only one will be shown."
       );
     }
+
+    const onAddDevice = async () => {
```

Comment:
following block of code is just copy/paste right?

Reply:
Not only. I extracted the `addDevice` and `addRecoveryPhrase` because I need them in the `determineTempKeysWarning` function.

The `determineTempKeysWarning` function is new.